src/components/Navigation/ExploreNavigation/index.js

```diff
@@ -101,10 +101,12 @@ function ExploreNavigation({ logoProps, ...props }) {
 
 ExploreNavigation.propTypes = {
   logoProps: PropTypes.shape({}),
+  handleTooltipOpen: PropTypes.func,
```

Comment:
It's a handle for event so we should name it accordingly: `onClick`, `onOpenHelp`, etc. `handleXXX` naming is used in the component handling the event.